src/test/resources/login-special-character-charset-CP1047.txt

```diff
@@ -0,0 +1,24 @@
+                                                                                
+       _______________________________________                                  
+      ¨GROUPAMA        ¨                      ¨           S y s t } m e s       
```

Comment:
I guess there is no problem on adding this open source project the name of the company, the name of the machine and terminal name. If there is I would advice you changing this file and associated dump, if you want, we can help you with that and change this as if you never uploaded this.

Reply:
Thank you for asking, but there is no problem indeed, I asked to my security officer if I can upload this, he tells me "no problem" so no problem for me :) 
Joke aside, there is no sensitive data inside and I replace all IP, so there is really no issue. 
However if that bother you to have some other company name in your repo, tell me and  I'll change the screen file and the dump.

Reply:
Not at all, just wanted to make sure. I will merge this right away, Thank you very much for this contribution. We hope you can keep contributing with such kind of features and good code.

Reply:
Thanks, but you have done almost all the work, I only brought the idea and the tests :)
In French we have a proverb for that, translated, it says " Return to Caesar what belongs to Caesar", it's a bit like "give credit where credit is due", so all credit for you :) .
Besides this, it was a pleasure to contribute, hopefully this will help others, and at least, it really do help me a lot